FATN_to_USB/fatn_to_usb/scrolling_text.py

```
@@ -8,42 +8,32 @@
-  )
+# Try to import and initialize ST7789 display
+try:
+    import ST7789 as ST7789
+    disp = ST7789.ST7789(
+        port=0,
+        cs=ST7789.BG_SPI_CS_FRONT,  # BG_SPI_CSB_BACK or BG_SPI_CS_FRONT
+        dc=9,
+        backlight=19,               # 18 for back BG slot, 19 for front BG slot.
+        spi_speed_hz=80 * 1000 * 1000
+    )
 except BaseException as exception:
```

Ruff (0.14.8)

Do not catch blind exception: `BaseException`

(BLE001)